tabs/src/components/sample/Monaco.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import MonacoEditor from 'react-monaco-editor';
+import PropTypes from 'prop-types';
+
+
+const Monaco = ( props ) => {
+  // eslint-disable-next-line react/prop-types
```

Comment:
Does introducing prototypes prevent you from disabling it? I've previously used `import PropTypes from 'prop-types';` to resolve this

Reply:
On line 3, I have imported Proptypes and specified the proptypes for the component below. But the error is still there

Reply:
you have to define proptypes like this
```
Monoco.propTypes = {
  body: PropTypes.string
... etc
};
```

Reply:
Yes thank you so much!